sentry.server.config.ts

```
@@ -2,12 +2,13 @@
 // https://docs.sentry.io/platforms/javascript/guides/nextjs/
 
+import { NEXT_PUBLIC_SENTRY_DSN, NODE_ENV, SENTRY_TRACES_SAMPLE_RATE } from "@/config";
 import * as Sentry from "@sentry/nextjs";
 
-if (process.env.NODE_ENV === 'production') {
+if (NODE_ENV === 'production') {
   Sentry.init({
-    dsn: process.env.NEXT_PUBLIC_SENTRY_DSN,
-    tracesSampleRate: parseFloat(process.env.SENTRY_TRACES_SAMPLE_RATE as string) ?? 0.1,
+    dsn: NEXT_PUBLIC_SENTRY_DSN,
+    tracesSampleRate: parseFloat(SENTRY_TRACES_SAMPLE_RATE) ?? 0.1,
```

Biome (1.9.4)

[error] Use Number.parseFloat instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseFloat instead.


(lint/style/useNumberNamespace)